exonum/src/node/mod.rs

```diff
@@ -439,7 +441,18 @@ impl NodeHandler {
             system_state.current_time(),
         );
 
-        let is_enabled = api_state.clone().is_enabled();
+        let mut is_enabled = api_state.clone().is_enabled();
+        let is_validator = validator_id.is_some();
+
+        if !is_validator {
+            if is_enabled {
+                error!("Provided enabled consensus for auditor node")
```

Comment:
Error message is not clear
"Consensus is enabled, but the current node is auditor"